app/src/organisms/ChooseRobotSlideout/AvailableRobotOption.tsx

```diff
@@ -20,44 +24,86 @@ interface AvailableRobotOptionProps {
 }
 
 export function AvailableRobotOption(
   props: AvailableRobotOptionProps
 ): JSX.Element {
-  const { robotName, robotModel, local, onClick, isSelected } = props
+  const {
+    robotName,
+    robotModel,
+    local,
+    onClick,
+    isSelected,
+    isOnDifferentSoftwareVersion,
+  } = props
+  const { t } = useTranslation('protocol_list')
   return (
-    <MiniCard onClick={onClick} isSelected={isSelected}>
-      <img
-        src={OT2_PNG}
-        css={css`
-          width: 6rem;
-        `}
-      />
-      <Flex
-        flexDirection={DIRECTION_COLUMN}
-        marginLeft={SPACING.spacing4}
-        marginTop={SPACING.spacing3}
+    <>
+      <MiniCard
+        onClick={onClick}
+        isSelected={isSelected}
+        isNonviable={isOnDifferentSoftwareVersion && isSelected}
       >
-        <StyledText as="h6">{robotModel}</StyledText>
-        <Box maxWidth="9.5rem">
-          <StyledText
-            as="p"
-            css={{ 'overflow-wrap': 'break-word' }}
-            fontWeight={TYPOGRAPHY.fontWeightSemiBold}
-          >
-            {robotName}
-            <Icon
-              // local boolean corresponds to a wired usb connection
-              aria-label={local ? 'usb' : 'wifi'}
-              marginBottom={`-${SPACING.spacing2}`}
-              marginLeft={SPACING.spacing3}
-              name={local ? 'usb' : 'wifi'}
-              size={SIZE_1}
-            />
-          </StyledText>
-        </Box>
-      </Flex>
-    </MiniCard>
+        <img
+          src={OT2_PNG}
+          css={css`
+            width: 6rem;
+          `}
+        />
+        <Flex
+          flexDirection={DIRECTION_COLUMN}
+          marginLeft={SPACING.spacing4}
+          marginTop={SPACING.spacing3}
+        >
+          <StyledText as="h6">{robotModel}</StyledText>
+          <Box maxWidth="9.5rem">
+            <StyledText
+              as="p"
+              css={{ 'overflow-wrap': 'break-word' }}
+              fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+            >
+              {robotName}
+              <Icon
+                // local boolean corresponds to a wired usb connection
+                aria-label={local ? 'usb' : 'wifi'}
+                marginBottom={`-${SPACING.spacing2}`}
+                marginLeft={SPACING.spacing3}
+                name={local ? 'usb' : 'wifi'}
+                size={SIZE_1}
+              />
+            </StyledText>
+          </Box>
+        </Flex>
+        {isOnDifferentSoftwareVersion && isSelected ? (
+          <>
+            <Box flex="1 1 auto" />
+            <Icon name="ot-alert" size="1.25rem" color={COLORS.error} />
+          </>
+        ) : null}
+      </MiniCard>
+
+      {isOnDifferentSoftwareVersion && isSelected ? (
+        <StyledText
+          as="label"
+          color={COLORS.errorText}
+          css={css`
+            & > a {
+              color: ${COLORS.errorText};
+              text-decoration: ${TEXT_DECORATION_UNDERLINE};
```

Comment:
```suggestion
              text-decoration: ${TYPOGRAPHY.textDecorationUnderline};
```
We can remove `TEXT_DECORATION_UNDERLINE` (L15)